### DISCUSSION TITLE
Plugin installtion issue in version 5.2.0

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#514] [Installation] [answered]
Hi Team!
In jenkins setup Many of the plugin installtions are failed , While running Install suggested plugin . below are the list of plugins.
@santhosh-tg Kindly check and suggest.

Pipeline: API

Timestamper

Script Security

JavaBeans Activation Framework (JAF) API

JAXB

Jackson 2 API

ECharts API

Display URL API

Pipeline: Supporting APIs

Checks API

JUnit

Matrix Project

Workspace Cleanup

Pipeline: Nodes and Processes

Command Agent Launcher

Oracle Java SE Development Kit Installer

Pipeline: Groovy

Pipeline: Job

Jakarta Activation API

Jakarta Mail API

Mailer

Pipeline: Basic Steps

Gradle

Pipeline: Milestone Step

Pipeline: Build Step

Pipeline: Groovy Libraries

Pipeline: Stage Step

Pipeline: Model API

Pipeline: Declarative Extension Points API

Branch API

Pipeline: Multibranch

Pipeline: Stage Tags Metadata

Git client

Pipeline: Input Step

Pipeline: Declarative

Pipeline

Java JSON Web Token (JJWT)

GitHub API

Git

GitHub

GitHub Branch Source

Pipeline: GitHub Groovy Libraries

Pipeline Graph Analysis

Pipeline: REST API

Pipeline: Stage View
Email Extension

LDAP

![Screenshot 2023-06-06 173017](https://github.com/Sunbird-Ed/Community/assets/41056032/0f5a6f21-031f-4bf3-b8fc-fe2a0cb5a0b9)

**Comment by chitranshu-keshav** [comment]
@santhosh-tg @tsprasath can you please get back on this

**Reply by tsprasath** [reply]
@gohilamariappan Can you give retry and check it. Also please ensure following steps are carried out.

- Open Jenkins UI in a browser by visiting JENKINS_IP:8080
- Enter the initial password and follow the on-screen instructions. Choose Install suggested plugin and create an admin user
- Go to http://JENKINS_IP:8080/pluginManager/available -> Search for ‘Configuration as Code Plugin’ and install the plugin without restart.

**Reply by heungheung** [reply]
@tsprasath IMHO, it probably won't work. 

2.319.3 LTS is pretty old..... the team should consider move to 2.346.3 if JDK8 is a must.... 

I tried in different environments & OSes, only Folders, OWASP Markup Formatter & Matrix AuthZ plugin can be installed - all other plugin will fail to install..... 

Again, 2.319.3 is just too old. Right now, if I have to stay with 2.319.3, I will try to tgz all the plugin I have in an old environment and put them in a new jenkins installation.

**Reply by heungheung** [reply]
BTW & FYI, I tried 2.346.3 half year back.... not sure if the Jenkins part (I mean Jenkins install, plugin install & job loading) will still work without any problem now.

In terms of (Sunbird-Ed) build, artifact upload, service provision & deploy, 2.346.3 works fine without any problem. Will be good if other community members can also share their experience.

**Reply by santhosh-tg** [reply, marked as answer]
@gohilamariappan  As you have mentioned in the other thread [512](https://github.com/orgs/Sunbird-Ed/discussions/512) I take it that installing jenkins version `2.346.3` fixed this issue. Please confirm and close the thread if its resolved.

**Comment by gohilamariappan** [comment]
@santhosh-tg  Issue resolved .  
 [512](https://github.com/orgs/Sunbird-Ed/discussions/512)
